lib/chat_api_web/router.ex

```diff
@@ -102,6 +102,7 @@ defmodule ChatApiWeb.Router do
     get("/google/auth", GoogleController, :auth)
     get("/google/oauth", GoogleController, :callback)
     get("/google/authorization", GoogleController, :authorization)
+    delete("/google/authorization/:id", GoogleController, :delete)
```

Comment:
not a blocker but might be nice to have a test for this (like https://github.com/papercups-io/papercups/blob/master/test/chat_api_web/controllers/slack_controller_test.exs#L64-L81) if you have some time later!